### ISSUE TITLE
Can the project be taken to Google summer of code 2021 as organization

### ISSUE THREAD

**Issue by pawan-nirpal-031** [bluespec/Piccolo#37]
Hello to people maintaining the project, I would like to contribute to this amazing hardware project and I was wondering if this project can be taken to google summer of code 2021, the organization application openings start from February I believe, please do consider my request.

**Comment by charlie-bluespec** [comment]
@pawan-nirpal-031 
Yes, you can take this project to the google summer of code 2021.
You can use this project as you would any other open source project licensed under the permissive Apache 2.0 license.
Let us know if there are any other details you need clarified.
